src/databricks/labs/blueprint/entrypoint.py

```diff
@@ -41,11 +41,15 @@ def find_project_root(__file: str) -> Path:
     Idiomatic usage is: find_project_root(__file__)
     """
     this_path = Path(__file)
-    # TODO: detect when in wheel seems to be more challenging and pkgutil.get_data() might be necessary
+
     for leaf in ("pyproject.toml", "setup.py"):
         root = find_dir_with_leaf(this_path, leaf)
         if root is not None:
             return root
+    # Wheel installation is under site-packages hence return the parent of site-packages
+    if "site-packages" in __file:
+        return Path(__file.split("site-packages")[0] + "site-packages/")
```

Comment:
And what if there are many modules with "__about__.py"?

Reply:
This is presenting a workaround since we don't do anything with PROJECT_ROOT variables, to answer your question it doesn't matter, the path is going to be set from Python env location where wheel file is installed. 

Ideally just rewriting the get_logger to get module name directly would make sense.

Reply:
Please take a look at the usages of this function in the library.

As for remoph - logging.get_logger(__name__) might be enough. The utility of blueprint is to help only when __name__ is __main__